.github/workflows/test.yml

```
@@ -43,12 +43,13 @@ jobs:
         id: test
         env:
           RUN_EXTERNAL_KERNEL_MARK: ${{ contains(matrix.os, 'ubuntu') && matrix.python-version == '3.9' }}
+          RUN_DOWNLOAD_TEST_DATA: ${{ contains(matrix.os, 'ubuntu') && matrix.python-version == '3.9' }}
```

Just do one environment variable here and remove the or below.
```suggestion
          USE_EXTERNAL_DATA: ${{ contains(matrix.os, 'ubuntu') && matrix.python-version == '3.9' }}
```